plugins/catalog-graph/src/components/CatalogGraphPage/CatalogGraphPage.tsx

```diff
@@ -103,20 +104,22 @@ const useStyles = makeStyles(
   { name: 'PluginCatalogGraphCatalogGraphPage' },
 );
 
-export const CatalogGraphPage = (props: {
-  relationPairs?: RelationPairs;
-  initialState?: {
-    selectedRelations?: string[];
-    selectedKinds?: string[];
-    rootEntityRefs?: string[];
-    maxDepth?: number;
-    unidirectional?: boolean;
-    mergeRelations?: boolean;
-    direction?: Direction;
-    showFilters?: boolean;
-    curve?: 'curveStepBefore' | 'curveMonotoneX';
-  };
-}) => {
+export const CatalogGraphPage = (
+  props: {
+    relationPairs?: RelationPairs;
```

Comment:
nit: Can we remove this as it's exposed from the EntityRelationsGraphProps already?

Reply:
Done